portfolio/src/main/webapp/gallery.html

```diff
@@ -25,12 +25,22 @@
         <div class="name-bar-item">Rania Rusly</div>
       </nav>
 
-      <div class="section">
-        <h1>Cities through my lense!</h1>
+      <div class="map-section">
+        <h1>Cities through my lens!</h1>
         <p>Through the years, I like to take pictures of cities that I have been</p>
         <p>To look at the photo galleries, navigate through the map and select a pin!</p>
+        <p>This feature is a Work in Progress for now, I am prioritizing to try out the other API and libraries</p>
```

Comment:
nit: why not a single paragraph then? 🤔
same below

Reply:
Thank you for the suggestions :D by a single paragraph, do you mean combining line 30 31 and 32?

Reply:
Yes :D with only one `<p>` tag